python/paddle/fluid/executor.py

```diff
@@ -625,16 +635,14 @@ def _run_parallel(self, program, scope, feed, fetch_list, fetch_var_name,
             feed_tensor_dict = dict()
             for feed_name in feed:
                 feed_tensor = feed[feed_name]
+                var = global_block.var(feed_name) if need_check_feed else None
```

Comment:
What if `need_check_feed` is False? Then the conversation from `double` -> `float` would not happen?

Reply:
When running CompiledProgram, `need_check_feed==False` means the CompiledProgram is not constructed from graph instead of program. NOT sure if we need to support that case.